al_aws_collector.js

```diff
@@ -378,6 +402,28 @@ class AlAwsCollector {
             }
         ],
         function(err, checkinParts) {
+
+            let invocationStatsDatapoints = checkinParts[1].statistics[0].Datapoints;
+            let errorStatsDatapoints = checkinParts[1].statistics[1].Datapoints ;
+
+            if (checkinParts[0].status === 'ok' &&  invocationStatsDatapoints && invocationStatsDatapoints.length > 0 &&  errorStatsDatapoints && errorStatsDatapoints.length > 0 && errorStatsDatapoints[0].Sum === 0) {
```

Comment:
Don't we expect `errorStatsDatapoints.length` to be `0`?
Please refactor this line becasue it is too long.

Reply:
I have refactor the code and we will get datapoint array ([ { Timestamp: 2021-01-27T16:33:00.000Z, Sum: 0, Unit: 'Count' } ]) so checking Sum value will be 0 if there is no error.